packages/batch-poster-monitor/index.ts

```diff
@@ -221,23 +219,31 @@ const getBatchPosterLowBalanceAlertMessage = async (
     }
   }
 
-  const balance = await parentChainClient.getBalance({
+  const currentBalance = await parentChainClient.getBalance({
     address: batchPoster,
   })
 
-  const lowBalanceDetected =
-    (childChainInformation.parentChainId === 1 &&
-      balance < BigInt(LOW_ETH_BALANCE_THRESHOLD_ETHEREUM * 1e18)) ||
-    (childChainInformation.parentChainId !== 1 &&
-      balance < BigInt(LOW_ETH_BALANCE_THRESHOLD_ARBITRUM * 1e18))
+  // get the gas used in the last 24 hours
+  let gasUsedInLast24Hours = BigInt(0)
+  for (const log of sequencerInboxLogs) {
+    gasUsedInLast24Hours = (
```

Comment:
```suggestion
    gasUsedInLast24Hours += (
```
If I read it correctly